.github/workflows/install-shared-dependencies/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
           shell: bash
           if: "${{ inputs.os == 'macos-latest' }}"
           run: |
-              brew install git gcc pkgconfig openssl redis
+              brew install git gcc pkgconfig openssl redis coreutils
```

Comment:
what is this for?

Reply:
For `realpath` which is used by `install_and_test.sh`